docs/installation.md

```diff
@@ -21,7 +21,16 @@ Portfolio Performance ist nicht mit einem Apple Zertifikat signiert. Aus diesem
 
 Portfolio Performance nutzt [Eclipse SWT](https://www.eclipse.org/swt/) und damit unter Linux native Bibliotheken. Standardmässig wird dabei **GTK3** verwendet, allerdings gibt es bei verschiedenen Kombinationen mit Themes immer mal wieder Probleme. So meldete zum Beispiel ein Benutzer Probleme im Zusammenhang mit ["oxygen-gtk" Theme](https://github.com/buchen/portfolio/issues/1089#issuecomment-459698493).
 
-Die notwendigen Abhängigkeiten lassen sich unter Ubuntu 18.04 mit `sudo apt install libwebkitgtk-3.0-0 default-jre` installieren.
+Die notwendigen Abhängigkeiten sind bei aktuellen Linux-Distros schon vorhanden:
+```
+$ dpkg -l | grep libwebkit
+ii  libwebkit2gtk-4.0-37:amd64             2.38.5-0ubuntu0.22.04.1                     amd64        Web content engine library for GTK
+```
```

Comment:
Ich würde in einer Installationsanleitung für Anfänger nicht erläutern, was man *nicht* zu machen braucht. Leute werden außerdem denken, dass sie diesen Befehl ausführen sollen.

Ich würde also diesen Teil ganz weglassen.

Reply:
Die Argumentation kann ich nachvollziehen.